package.json

```diff
@@ -35,7 +38,10 @@
     "friendly-errors-webpack-plugin": "1.7.0",
     "html-webpack-plugin": "3.2.0",
     "jest": "23.6.0",
+    "postcss-loader": "3.0.0",
     "react-hot-loader": "4.6.3",
+    "style-loader": "0.23.1",
+    "tailwindcss": "0.7.3",
```

Comment:
dev dep?

Reply:
[See Discord] Only need it to process files during dev/build.